compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/KernelDispatch.cpp

```diff
@@ -168,7 +169,8 @@ FailureOr<ParameterSetting> ParameterSetting::create(linalg::LinalgOp linalgOp,
   // the element types of all tensors which need to be allocated in memory
   // simultaneously.
   FailureOr<unsigned> maybeScaleFactor =
-      getTilingScaleFactor(lhsType.getElementType());
+      isObjectFifo ? getTilingScaleFactor(initType.getElementType())
+                   : getTilingScaleFactor(lhsType.getElementType());
```

Comment:
I think we had the same question previously that why the lhsType is taken for other pipelines. I think I tried to use the initType before but got failure for ukernel.